src/forms/output/BenefitsTable.js

```diff
@@ -129,60 +130,60 @@ const BenefitsTable = function (props) {
         <Table.Cell
           textAlign='right'
           width={ 3 }
-          style={ totalsRowHeaderStyle }>Total Benefits
+          style={ totalsRowHeaderStyle }>{ snippets.rowTotalBenefits_v1 }
         </Table.Cell>
         <Table.Cell
           textAlign='right'
           width={ 3 }
-          style={ totalsRowStyle }>${totalBenefitCurrent} / month
+          style={ totalsRowStyle }>{ snippets.dollarSign_v1 }{totalBenefitCurrent}{ snippets.perMonth_v1 }
         </Table.Cell>
         <Table.Cell
           textAlign='right'
           width={ 3 }
-          style={ totalsRowStyle }>${totalBenefitFuture} / month
+          style={ totalsRowStyle }>{ snippets.dollarSign_v1 }{totalBenefitFuture}{ snippets.perMonth_v1 }
         </Table.Cell>
         <Table.Cell
           textAlign='right'
           width={ 3 }
-          style={ totalsRowStyle }>{ getSignSymbol(totalDiff) } ${ Math.abs(totalDiff) } / month
+          style={ totalsRowStyle }>{ getSignSymbol(totalDiff) } { snippets.dollarSign_v1 }{ Math.abs(totalDiff) }{ snippets.perMonth_v1 }
         </Table.Cell>
       </Table.Row>
     );
   };
 
-  const IncomeRow = function (props) {
+  const IncomeRow = function ({ snippets }) {
     return (
       <Table.Row>
-        <Table.Cell style={ rowHeaderStyle }>Income</Table.Cell>
-        <Table.Cell textAlign='right'>${incomeCurrent} / month</Table.Cell>
-        <Table.Cell textAlign='right'>${incomeFuture} / month</Table.Cell>
-        <Table.Cell textAlign='right'>{ getSignSymbol(incomeDiff) } ${ Math.abs(incomeDiff) } / month</Table.Cell>
+        <Table.Cell style={ rowHeaderStyle }>{ snippets.rowIncome_v1 }</Table.Cell>
+        <Table.Cell textAlign='right'>{ snippets.dollarSign_v1 }{incomeCurrent}{ snippets.perMonth_v1 }</Table.Cell>
+        <Table.Cell textAlign='right'>{ snippets.dollarSign_v1 }{incomeFuture}{ snippets.perMonth_v1 }</Table.Cell>
+        <Table.Cell textAlign='right'>{ getSignSymbol(incomeDiff) } { snippets.dollarSign_v1 }{ Math.abs(incomeDiff) }{ snippets.perMonth_v1 }</Table.Cell>
       </Table.Row>
     );
   };
 
-  const TotalsRow = function (props) {
+  const TotalsRow = function ({ snippets }) {
     return (
       <Table.Row style={{ border: 'none' }}>
         <Table.Cell
           textAlign='right'
           width={ 3 }
-          style={ totalsRowHeaderStyle }>Net Total
+          style={ totalsRowHeaderStyle }>{ snippets.rowNetTotal_v1}
```

Comment:
Looking great to me! May want to `npm run lint` to clean up a couple of things (does that work with JSX?).

Would also appreciate other opinions.

Reply:
npm run lint didn't make any changes to this PR.

Reply:
Hmm, I guess it doesn't do much with JSX. Maybe you'll have to edit this one manually. I know we were having trouble getting it to behave properly in that context or something.